raiden/tests/fuzz/test_state_changes.py

```diff
@@ -476,6 +529,12 @@ def update(self, amount, lock):
             )
 
 
+@dataclass
+class WithOurAddress:
+    our_address: typing.Address
+    data: typing.Any
```

Comment:
This is always an action, isn't it? We could name and type it that way. Or do you think we will use it in more generic cases soon?

Reply:
It will probably always be an action.